routing/router.go

```diff
@@ -854,6 +854,7 @@ func (r *ChannelRouter) syncGraphWithChain() error {
 
 		// Using the next height, request a manual block pruning from
 		// the chainview for the particular block hash.
+		log.Infof("Filtering block at height: %v", int64(nextHeight))
```

Comment:
Should we give a bit more info on what exactly this is doing? Otherwise during normal operations, this is a bit useless. So perhaps something like "Filtering blocks for closed channels, next height: %v"?

Reply:
I updated this, with a similar message to your suggestion